.prettierignore

```diff
@@ -4,3 +4,4 @@ dist/
 
 # Specific to prettier
 package.json
+CHANGELOG.md
```

Comment:
Any reason for not formatting this?

Reply:
it's automatically created by release. it's will make some noise when we want to format